### ISSUE TITLE
Disable anchor (i.e., fragment) on lvl0 / lvl1 entry

### ISSUE THREAD

**Issue by mojavelinux** [algolia/docsearch-configs#514]
Is it possible to disable the anchor on the lvl0 and lvl1 entry? My selectors look like this:

```
"lvl0": ".navbar .component",
"lvl1": ".doc > h1.page",
```

The lvl0 selector matches the following HTML:

```
<a class="navbar-link component" href="index.html">
<span class="title">Product Name</span>
<span class="version">5.5</span>
</a>
```

This link is contained within an ancestor that has the ID "topbar-menu". But notice there is no fragment on the href value itself.

The URL for the entry in the search index is showing up as:

    https://docs.example.org/product/index.html#topbar-menu

I don't want the `#topbar-menu` fragment to be included in the URL. The fragment is not in the URL that the scraper finds, so I don't know why it's being added. It should just link to the URL it finds. It's as though the scraper is looking upwards to find an ID. Is there anyway to get the scraper to not add the fragment to the URL in the index?

**Comment by mojavelinux** [comment]
This may just be on the lvl1 entry, which only makes me more confused since that's not even coming from the topbar. I don't see records that only contain a lvl0 entry (or I can't figure out how to search for them).

**Comment by mojavelinux** [comment]
I think I can answer my own question. This is where the `global` flag on the selector comes in. If I set the `global` flag to true, then the fragment is not added to the URL (if otherwise missing). But fragments are still added for section headings (i.e., deep links).

I also figured out that I need to set `min_indexed_level` to 1 so that a separate entry is not created for the lvl0 selector, which in my case is the site category (i.e., product). This cuts out a lot of duplicate results.

Here's the relevant part of my revised config:

```
  "selectors": {
    "lvl0": {
      "selector": ".navbar .component",
      "global": true
    },
    "lvl1": ".doc > h1.page",
    "lvl2": ".doc .sect1 > h2:first-child",
    "lvl3": ".doc .sect2 > h3:first-child"
  },
  "min_indexed_level": 1,
```
